### ISSUE TITLE
Why do you have two model code files in “models”?

### ISSUE THREAD

**Issue by davimedio01** [wgcban/adamae#5]
What's the difference between [“pre-training”](https://github.com/wgcban/adamae/blob/main/models/pretrain/vit.py) and [“fine-tuning”](https://github.com/wgcban/adamae/blob/main/models/finetune/vit.py) models? Shouldn't they be the same model code?

Taking advantage of the question, I would like to carry out a Continuous Training for the ASL Citizen base (classification task, like in K400), starting from the Pre-Training carried out on the K400, and then fine-tuning this trained model on a personal database. Should I carry out both processes (pre-training and fine-tuning) contained in the repository, or just fine-tune twice (once for each base, in sequence)?

**Comment by davimedio01** [comment]
I found my problem. In my case, I'm going to do “two” fine-tunings of the model, but since they're for the same task (just by changing the dataset), it turns out that using the model for fine-tuning will be correct, since it's already pre-trained in the k400.

However, I have a question about how to use Deepspeed correctly, since I want to use the class for the ["fine-tuning model"](https://github.com/wgcban/adamae/blob/main/models/finetune/vit.py) in my custom code, but I don't know how to start Deepspeed correctly in this case.
